src/ResourceManager/Network/Commands.Network.Test/ScenarioTests/CortexTests.cs

```diff
@@ -45,5 +46,15 @@ public void TestP2SCortexCRUD()
         {
             NetworkResourcesController.NewInstance.RunPsTest(_logger, "Test-P2SCortexCRUD");
         }
```

Comment:
This looks great! Only one small comment - please add the 
```
[Trait(Category.AcceptanceType, Category.CheckIn)]
```
to the TestPS2CortexCRUD test.

Reply:
@ritwikbasu Can you help adding this? Thanks!

Reply:
Added category type to P2S crud.